src/lib.rs

```
@@ -46,6 +67,200 @@ impl UniffiCustomTypeConverter for Uuid {
     }
 }
 
+pub struct StatusLists;
+
+impl TypedStatusMapProvider<Uri, BitstringStatusListCredential> for StatusLists {
+    async fn get_typed(&self, id: &Uri) -> Result<MaybeCached<StatusList>, ProviderError> {
+        eprintln!("fetch <{id}>");
+        Ok(MaybeCached::NotCached(StatusList::from_bytes(
+            1.try_into().unwrap(),
```

let's return an error if this conversion fails